dojo/notifications/helper.py

```diff
@@ -73,16 +83,26 @@ def create_notification(event=None, **kwargs):
                 .filter((Q(applicable_notifications_count__gt=0) | Q(is_superuser=True)))
 
             # only send to authorized users or admin/superusers
+            logger.debug('Filtering users for the product %s', product)
+
             if product:
                 users = get_authorized_users_for_product_and_product_type(users, product, Permissions.Product_View)
+
             elif product_type:
                 users = get_authorized_users_for_product_type(users, product_type, Permissions.Product_Type_View)
+            else:
+                # nor product_type nor product defined, we should not make noise and send only notifications to admins
+                logger.debug('Product is not specified, making it silent')
+                users = users.filter(is_superuser=True)
+
```

Comment:
Here is an actual flake8 error, hidden so far by the git problem

Reply:
fixed